BFWControls/Modules/Carousel/Controller/CarouselViewController.swift

```diff
@@ -234,7 +236,7 @@ extension CarouselViewController {
     
     open override func scrollViewDidScroll(_ scrollView: UIScrollView) {
         if scrollView == collectionView {
-            updatePageControl()
+            updatePageControl(collectionViewSize ?? CGSize() == scrollView.frame.size)
```

Comment:
Don't instantiate a `CGSize` just for temporary comparison. Is your intention to default to a zero size? If so then perhaps you want:

```swift
updatePageControl(shouldUpdateCurrentPage: (collectionViewSize ?? CGSize.zero) == scrollView.frame.size)
```

However, I suppose it's more accurate to just not try to update the current page if `collectionViewSize` is nil, eg:

```swift
updatePageControl(shouldUpdateCurrentPage: collectionViewSize.map { $0 == scrollView.frame.size } ?? false )
```